### PR TITLE
Installation Overview: Provide descriptive link text

### PR DESCRIPTION
**Pull request by Rllopez0410**
Updated text surrounding the link with the given text in the excel sheet.

## Because
Update the text in the the Installation Overview lesson that surround the links with the the preferred content that was given via excel. 

## This PR
- Removed the wiki link related to OS.
- Modified other text surrounding the links with the requested changes given via excel.
- Removed the word <code>this</code> within the text surrounding the VM link.

## Issue
Related to [#27681](https://github.com/TheOdinProject/curriculum/issues/27681)

## Pull Request Requirements

-   [x] I have thoroughly read and understand [The Odin Project Contributing Guide](https://github.com/TheOdinProject/.github/blob/main/CONTRIBUTING.md)
-   [x] The title of this PR follows the `location of change: brief description of change` format, e.g. `Intro to HTML and CSS lesson: Fix link text`
-   [x] The `Because` section summarizes the reason for this PR
-   [x] The `This PR` section has a bullet point list describing the changes in this PR
-   [x] If this PR addresses an open issue, it is linked in the `Issue` section
-   [x] If any lesson files are included in this PR, they have been previewed with the [Markdown preview tool](https://www.theodinproject.com/lessons/preview) to ensure it is formatted correctly
-   [x] If any lesson files are included in this PR, they follow the [Layout Style Guide](https://github.com/TheOdinProject/curriculum/blob/main/LAYOUT_STYLE_GUIDE.md)

### CODE REVIEW

File: foundations/installations/installation_overview.md [hunk 1]
```diff
@@ -45,7 +45,7 @@ Windows, by itself, **is not natively supported** by The Odin Project, or on our
 
 A **virtual machine** is an emulation of a computer that runs within your existing OS. It allows you to use another operating system inside of a program on your current operating system (e.g. Running Linux inside of Windows). Virtual machines are as straightforward to install as any other program and are risk free. If you don't like Linux, you can easily remove the virtual machine. Virtual machines are a great way for new developers to get started quickly.
```
Comment (MaoShizhong) [inline foundations/installations/installation_overview.md:46]:
```suggestion
A [virtual machine](https://youtu.be/yIVXjl4SwVo) is an emulation of a computer that runs within your existing OS. It allows you to use another operating system inside of a program on your current operating system (e.g. Running Linux inside of Windows). Virtual machines are as straightforward to install as any other program and are risk free. If you don't like Linux, you can easily remove the virtual machine. Virtual machines are a great way for new developers to get started quickly.
```
Putting the link from line 48 here (and then deleting line 48 as it's no longer needed) will resolve the linting error **and** keep the link text sufficiently descriptive.

**Review by MaoShizhong — CHANGES_REQUESTED** [review]
Thank for getting this lesson sorted! Good catch on the VM video link which I missed. There's a change I've suggested which should resolve those issues.

File: foundations/installations/installation_overview.md [hunk 2]
```diff
@@ -45,7 +45,7 @@ Windows, by itself, **is not natively supported** by The Odin Project, or on our
...
 
-- [Watch this video](https://youtu.be/yIVXjl4SwVo) on Virtual Machines to achieve an overview of how they work.
+- [A video](https://youtu.be/yIVXjl4SwVo) on Virtual Machines to achieve a general understanding of how they work.
```
[the remark this answers is not in the document]
Comment (Rllopez0410) [reply foundations/installations/installation_overview.md:48]: that sounds perfect; i was not thinking about how a screen reader would read through the line.

**Review by MaoShizhong — CHANGES_REQUESTED** [review]
Thanks for moving the link up. Could you delete line 48 (the bullet point) since it's no longer needed?

**Comment by Rllopez0410** [comment]
Of course, I didn't realize it was still there. Line 48 has been removed.

**Review by MaoShizhong — APPROVED** [review]

**Pull request merged**